steam-client/src/main/java/com/avenga/steamclient/util/retry/RetryHandlerUtil.java

```diff
@@ -0,0 +1,32 @@
+package com.avenga.steamclient.util.retry;
+
+import com.avenga.steamclient.exception.CallbackCompletionException;
+import com.avenga.steamclient.exception.CallbackTimeoutException;
+import com.avenga.steamclient.model.steam.SteamMessageCallback;
+import com.avenga.steamclient.steam.client.SteamClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.avenga.steamclient.constant.Constant.RETRY_EXCEPTION_MESSAGE_FORMAT;
+
+public class RetryHandlerUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RetryHandlerUtil.class);
+
+    public static <T, R> R getOrRetry(SupplierCallbackHandler<R> supplier, SteamMessageCallback<T> messageCallback,int retryCount,
+                                    SteamClient client) throws CallbackTimeoutException {
+        var count = 1;
+        var errorMessage = "";
```

Comment:
u don't need that here

Reply:
This error message will help identify which callback wasn't processed after retries, otherwise it will be need to check error stack trace.

Reply:
got it